GenderPayGap.Core/Global.cs

```diff
@@ -113,7 +113,7 @@ public static string DownloadsLocation
         public static Version Version => Misc.GetTopAssembly().GetName().Version;
         public static DateTime AssemblyDate => Misc.GetTopAssembly().GetAssemblyCreationTime();
         public static string AssemblyCopyright => Misc.GetTopAssembly().GetAssemblyCopyright();
-        public static string DatabaseConnectionName => Config.GetAppSetting("DatabaseConnectionName") ?? "GpgDatabase";
+        public static string DatabaseConnectionName => Config.GetAppSetting("DatabaseConnectionName") ?? "gpg_database";

```

Comment:
The fact you had to make this change makes me wonder if we're missing this setting in the `appsettings.json` file (maybe actually `appsettings.LOCAL.json`)?

Reply:
This change has now gone